join-calling-to-teams-meeting/package.json

```diff
@@ -10,8 +10,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/communication-calling": "^1.0.1-beta.1",
-    "@azure/communication-common": "^1.0.0"
+    "@azure/communication-calling": "^1.1.0",
```

Comment:
This needs to be latest beta please. Joining teams meeting is a beta feature.

Reply:
good catch! fixed. thanks :)